frontend/src/metabase/core/components/FormField/FormField.styled.tsx

```diff
@@ -24,13 +25,22 @@ export const FieldCaption = styled.div<FormCaptionProps>`
 
 export interface FieldLabelProps {
   hasError: boolean;
+  titleStyle?: "small" | "large";
 }
 
 export const FieldLabel = styled.label<FieldLabelProps>`
   display: block;
   color: ${props => (props.hasError ? color("error") : color("text-medium"))};
   font-size: 0.77rem;
   font-weight: 900;
+
+  ${props =>
+    props.titleStyle === "large" &&
+    css`
```

Comment:
I opt for another `css` block to not have to nest conditions on existing properties (only `color` in this case)